unlock-app/src/pages/about.js

```diff
@@ -18,6 +18,13 @@ import { TwitterTags } from '../components/page/TwitterTags'
 import OpenGraphTags from '../components/page/OpenGraphTags'
 
 const Stories = [
+  {
+    date: new Date('2018-12-14'),
+    summary:
+      'Using ads to support content creation is not working anymore, whether it is from a purely technical aspect when more and more of the provisioned ads are not being displayed, from an economic standpoint where, outside of a shrinking number of larges platforms, nobody is able to even scrape cents per hour spent staring at ads.',
```

Comment:
I love this article! Glad to see it linked here.

I'd prefer that we use newlines so it is easier to read the summary in the editor.

How about this for a summary (it's a bit shorter):

> Using ads to support content creation is broken, whether from a purely technical or an economic standpoint.  Outside of a shrinking number of larges platforms, nobody is able to even scrape cents per hour spent staring at ads. 

This may have lost some of the meaning you're looking for, so just checking that to be sure.

Reply:
+1 to @cellog's edit, and to using newlines.

Reply:
Thanks! Updated the text and addded newlines (including for previous posts!).